src/Tests/Microsoft.NET.Sdk.Razor.Tests/ScopedCssIntegrationTests.cs

```diff
@@ -95,13 +95,16 @@ public void CanOverrideScopeIdentifiers()
             File.Move(Path.Combine(projectDirectory.Path, "Components", "Pages", "Counter.razor.css"), styles);
 
             var build = new BuildCommand(projectDirectory);
-            build.Execute("/p:EnableDefaultScopedCssItems=false").Should().Pass();
+            build.Execute("/p:EnableDefaultScopedCssItems=false", "/p:EmitCompilerGeneratedFiles=true").Should().Pass();
 
             var intermediateOutputPath = Path.Combine(build.GetBaseIntermediateDirectory().ToString(), "Debug", DefaultTfm);
 
             var scoped = Path.Combine(intermediateOutputPath, "scopedcss", "Styles", "Pages", "Counter.rz.scp.css");
             new FileInfo(scoped).Should().Exist();
             new FileInfo(scoped).Should().Contain("b-overriden");
+            var generated = Path.Combine(intermediateOutputPath, "generated", "Microsoft.NET.Sdk.Razor.SourceGenerators", "Microsoft.NET.Sdk.Razor.SourceGenerators.RazorSourceGenerator", "_Components_Pages_Counter.razor.cs");	
```

Comment:
Note that we (roslyn) don't make any guarantees about the structure of the files we emit into the generated folder, and they may change in the future.

Reply:
Is this something we can workaround by setting `GeneratedFilesOutputPath` or will the changes affect the filename used and not the directory path?

Reply:
No, that just sets the root location of the `generated` directory. Everything under that is essentially implementation specific. We do have an internal method that gets the path (see http://sourceroslyn.io/#Microsoft.CodeAnalysis/SourceGeneration/GeneratorDriver.cs,394) that we could consider making public, but won't immediately help you here...

Reply:
You might be better off just checking for the existence of *something* in that directory, rather than a specific file.

Reply:
I see. In this case, we're particularly trying to make sure that the correct information is propagated into the generated code, instead of just verifying that the file exists.

Is there an alternative approach we can use to verify generated contents?

Reply:
Could we enumerate the directory and look for a file with specific content?

Reply:
That seems a little excessive for this particular scenario.

Perhaps this is something that would best be supported when we role out unit tests using the official source generator library....